### ISSUE TITLE
Put the color picker tool in the toolbox

### ISSUE THREAD

**Issue by Vectornaut** [glimpse-editor/Glimpse#483]
### Problem: I just spent twenty minutes looking for the color picker tool
As a longtime GNU I.M.P user and occasional Photoshop user, I'm used to finding the color picker tool in the toolbox. The first time I wanted to use the color picker tool in Glimpse, I spent a few minutes scanning the toolbox for it, gave up, and launched the GNU I.M.P instead. The second time, after fruitlessly scouring the toolbox and searching the internet for documentation, I started to wonder if Glimpse had gotten rid of the tool entirely. Finally, by luck, I found it in the *Tools* menu. Now I feel pretty dumb, but I also suspect I may not be the only person with this problem.

### Preferred solution
Put the color picker tool in the toolbox (the grid of tool icons in the upper left corner of the Glimpse window).

### Alternative solution
Publish documentation which affirms that Glimpse has a color picker tool, and explains how to find it in the *Tools* menu. For this solution to work, the documentation must be easy to find through web searches like

 * glimpse eyedropper
 * glimpse color picker tool
 * glimpse "color picker" tool
 * glimpse colour picker tool

**Comment by ryliejamesthomas** [comment]
It is in the toolbox.

Maybe you have your tools grouped (Edit>Preferences>Interface>Toolbox>...), if so it's in the same group as the Measure tool. Even so, I think it's the default tool in that group.

**Comment by ryliejamesthomas** [comment]
Not to dismiss your issue. Something must be up, or you would have seen it.

**Comment by TrechNex** [comment]
It would also be useful to know:

- Which version of Glimpse Image Editor are you running?
- Which operating system are you using? (Windows or Linux)
- If you're using Linux, which installation option did to pick?

There was a change in how the toolbar was arranged between 0.1.2 and 0.2.0 as we jumped from GNU I.M.P 2.10.12 to 2.10.18.

**Comment by Vectornaut** [comment]
Aha! Yes, I have tools grouped, as @ryliejamesthomas guessed. I distinctly recall noticing the drawer markers (the triangles at the bottom right corners of the icons) and trying to pull out the drawers while I was looking for the color picker. I don't think I ever succeeded in pulling out the drawers, because I also recall thinking *I guess they got rid of the bucket fill*, which is grouped with the gradient tool.

I was probably trying to pull out the drawers with a double-click, which instead highlights the tool options panel.

> Even so, I think it's the default tool in that group.

I seem to have *Save tool options on exit* selected, and I recall having used the measure tool in a previous Glimpse session. I'm almost certain that the measure tool was active when I was looking for the color picker. This means I must have managed to find the measure tool without a problem in a previous session!

@TrechNex, I just upgraded to Glimpse 0.2.0 on Ubuntu 20.04. The last time I couldn't find the color picker, I was running Ubuntu 18.04.

**Bottom line**

* I'm ready to chalk this up to unfamiliarity with UI conventions—specifically, failing to pull out the drawers, and then disregarding the drawer indicators because of that.
* Once @ryliejamesthomas pointed out that tools are grouped, I learned how to pull out the drawers very quickly, so I think online documentation would've been the best solution for me.
* Are there any efforts underway to port the GNU I.M.P documentation to Glimpse, so it describes the Glimpse interface and shows up in searches about Glimpse? I'd be glad to help out with that.

**Comment by TrechNex** [comment]
It would be amazing if we could port the GNU I.M.P docs to Glimpse, but in practise that is probably not going to happen because there is far too much of it.

Trust me as someone who does technical documentation as their day job, the initial port is the easy part. The maintaining and keeping it up to date is the tricky part!

What I have tried to do is add extra documentation here:
Wiki: https://github.com/glimpse-editor/Glimpse/wiki
FAQs page: https://glimpse-editor.github.io/about/

I am also hoping to make 0.3.0 (due July 2021) compatible with upstream's offline help pages, but I haven't looked at what's involved so not sure how feasible that would be.
